AlfrescoSDK/AlfrescoSDK/Session/OAuth/AlfrescoOAuthNSLoginViewController.m

```diff
@@ -258,6 +261,7 @@ - (void)webView:(WebView *)webView decidePolicyForNavigationAction:(NSDictionary
         else
```

Comment:
It looks like there's a logic check missing here, should this code read the same as the iOS implementation?
```objectivec
else if(self.cloudConnectionStatus == CloudConnectionStatusInactive)
```